Automatic Validation ended with: 
> 2026-04-09T21:17:56.9131758Z ##[error] Manifest Error: Schema header not found. File: 7a88c6ac6e815245720d6140446347651abe0a6ad553c9e3355e9d496307d446 Manifest Error: Schema header not found. File: ca1b7814ee3f4f63b609c1a58989e36a7f27a3f0185b85fdc76ada5593b8ceae Manifest Error: Schema header not found. File: fee2ea5c5c2c664c7414ede5ecaeeacf8932e7a26d2d446c022916abbdfb0966

(Automated response - build 1244.)

--- a/manifests/g/gorodulin/prj/0.4.1/gorodulin.prj.installer.yaml
+++ b/manifests/g/gorodulin/prj/0.4.1/gorodulin.prj.installer.yaml
@@ -1,3 +1,4 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
 PackageIdentifier: gorodulin.prj
 PackageVersion: 0.4.1
 InstallerType: portable

--- a/manifests/g/gorodulin/prj/0.4.1/gorodulin.prj.locale.en-US.yaml
+++ b/manifests/g/gorodulin/prj/0.4.1/gorodulin.prj.locale.en-US.yaml
@@ -1,3 +1,4 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
 PackageIdentifier: gorodulin.prj
 PackageVersion: 0.4.1
 PackageLocale: en-US

--- a/manifests/g/gorodulin/prj/0.4.1/gorodulin.prj.yaml
+++ b/manifests/g/gorodulin/prj/0.4.1/gorodulin.prj.yaml
@@ -1,3 +1,4 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
 PackageIdentifier: gorodulin.prj
 PackageVersion: 0.4.1
 DefaultLocale: en-US

